### DISCUSSION TITLE
[Suggestion] Water added to gameplay

### DISCUSSION THREAD

**Discussion by TheDcol** [sunflower-land/sunflower-land#3472] [Ideas]
_**This will probably be a controversial idea as it would add more management, thinking and planning in the game whilst also adding a semi-optional gold cost within the core gameplay. Based off https://github.com/sunflower-land/sunflower-land/discussions/2379**_

A way to introduce water as a resource needed to grow crops. This would add an intrinsinc gameplay burn into the game where players need water to grow their crops. A breakdown:

- A farm has water wells, each water well contains x water capacity.
- Planting crops uses up x water. x water used is depended on the base time the crop takes to grow. planting 1 sunflower would use MUCH LESS water than planting 1 eggplant.
- Water wells are replenished every x days. During times of drought less water is replenished, during times of heavy rainfall more water is replenished.
- Option for players to buy extra 'Water Tanks' to store more water.
- When a player runs out of water they can still plant crops but the time/yield will be worse. They can wait until water is replenished OR alternatively a player can purchase water from NPC by exchanging Gold or other resource.
- Weather and Seasons could also be introduced in tandem with this mechanic.
- Depending on the season, certain crops may require more or less water than others.
- Mechanic would start after first island or after the 1st water well is built. Care for new players must be looked at..

WEATHER:
Sunny, Cloudy and Rainy. Every day the weather could change. Visually, the farm could look different depending on the weather.

SEASONS:
Summer, Winter, Spring, Autumn. Every x month the season changes. Add different types of consequences depending on the season. If summer, crops might grow slightly quicker and provide more yield. In winter, crops could take longer to grow and yield less than usual. Option to have some crops to better/worse than others depending on the season.

EVENTS:
Introduce random events throughout the year such as 'drought events' where less water is replenished in the wells.. water becomes more precious and players need to manage accordingly. Alternatively have 'monsoon events' where it rains more frequently and wells are replenished more often. Option to have some crops to be better/worse than others depending on the season (could be combined with weather too.. Maybe Sunflowers grow quicker in summer AND when it is sunny, extra yield).

Challenges...
- Does it overcomplicate the gameplay? Do players just want a game where they click a few times without thinking?
- Does it add too much friction? 
- Should we abandon the idea of adding costs to the core gameplay and just build on 'pay for boosts' type of mechanics? 

The basis of this idea is to add some extra bit of thinking for players AND to be more inline with farming games where weather/seasons play a BIG role alongside water. It would be a drastic addition to the gameplay but one that personally would intrigue me and add to the farming element. It would also open the doors for future mechanics, items and events to be built on based on this idea. **Having said that, I also understand the challenges that would come with changing the core gameplay.**

**Comment by Dionis404** [comment]
I think mechanics like this could change our wells and make them upgradeable. 
Well > ....> .... > Water tower 
Which would increase water production

**Reply by TheDcol** [reply]
For sure, it would open many paths, obviously with a resource cost

**Reply by Dionis404** [reply]
And if there will be seasons, you can add the parameter "heat", and make the heating of the house and greenhouse. and will burn wood. 
But that could complicate the game a lot. 

I think it would be a good option to add such mechanics on new islands. 
Desert - water requirement and new wells 
Winter island - heating and stoves

**Comment by IceMonky29** [comment]
To this you could add a greenhouse NFT with a reduced space in which players could plant and harvest a crop that loses efficiency during a season, be it winter, summer or spring, but this could work and be even more useful if certain crops could have a chance of being damaged during a season that is not favorable to them, this will add more difficulty and playability, I like your proposal. Also if a calendar is added to the bumpkin plaza with the seasons it would be very cool

**Reply by TheDcol** [reply]
Absolutely it could be an option, water mechanic along with weather opens up a lot of paths for the game to evolve/add depth

**Reply by IceMonky29** [reply]
exactly that would make the game much more attractive

**Comment by rodolfoddias** [comment]
I love the concept, but I am not so sure about spending gold to buy water. SFL is the metaverse currency, so I believe we should still use it to buy things from other NPCs. 

But I am 100% in favor of using gold and other resources to upgrade the well and maintain it - repair concept that can also be related to seasons. Hurricanes happens during Harvest Moon's Summer Season and when it happens it can break your stuff / remove crops. Any broken stuff would need to be repaired

**Reply by TheDcol** [reply]
Ye we can use SFL to buy the water instead, i think it would be okay.

Heavily in favour of having to maintain your farm or repair it, using resources for it :)
